src/index.html

```diff
@@ -27,6 +30,14 @@
       >
         Profile
       </button>
+      <button
+        id="user-management-btn"
+        class="element-display-remove"
+        type="button"
+        onclick="location.href='user-management/index.html'"
```

Comment:
Why is this not an `<a>` tag?

Reply:
yes this should be an <a> tag, updated it